index.bs

```diff
@@ -376,42 +408,75 @@ To <dfn>parse an attribution destination</dfn> from a string |str|:
 
 <h3 algorithm id="obtaining-attribution-source-anchor">Obtaining an attribution source from an <code>a</code> element</h3>
 
-To <dfn>obtain an attribution source</dfn> from an <{a}> element |anchor|:
+To <dfn>obtain an attribution source from an anchor</dfn> given an <{a}> element |anchor|:
+
+1. Let |params| be an {{AttributionSourceParams}} with the following key/value pairs:
+     : {{AttributionSourceParams/attributionDestination}}
+     :: |anchor|'s <{a/attributiondestination}> attribute
+     : {{AttributionSourceParams/attributionSourceEventId}}
+     :: |anchor|'s <{a/attributionsourceeventid}> attribute
+     : {{AttributionSourceParams/attributionReportTo}}
+     :: |anchor|'s <{a/attributionreportto}> attribute
+     : {{AttributionSourceParams/attributionExpiry}}
+     :: |anchor|'s <{a/attributionexpiry}> attribute
+     : {{AttributionSourceParams/attributionSourcePriority}}
+     :: |anchor|'s <{a/attributionsourcepriority}> attribute
+1. Return the result of running [=obtain an attribution source from params=] with |params|.
+
+<h3 algorithm id="obtaining-event-attribution-source-anchor">Obtaining an event attribution source from an <code>a</code> element</h3>
+
+To <dfn>maybe obtain an event attribution source from an anchor</dfn> given an <{a}> element |anchor|, run the following steps:
+
+1. If |anchor| does not have a <{a/registerattributionsource}> attribute, return null.
+1. Let |source| be the result of running [=obtain an attribution source from an anchor=] with |anchor|.
+1. If |source| is null, return null.
+1. Return the result of running [=obtain an event attribution source from a source=] with |source|.
+
+<h3 algorithm id="obtaining-event-attribution-source-source">Obtaining an event attribution source from a source</h3>
+
+To <dfn>obtain an event attribution source from a source</dfn> given an [=attribution source=] |source|, run the following steps:
+
+1. Set |source|'s [=attribution source/source type=] to "<code>event</code>".
+1. Round |source|'s [=attribution source/expiry=] away from zero to the nearest day.
```

Comment:
Filed #236 for this.

Reply:
Actually, `expiry` is a duration, not a date, so I don't think there's any ambiguity in terms of time zone or leap seconds. Neither should apply, correct?

Reply:
https://pr-preview.s3.amazonaws.com/apasel422/conversion-measurement-api/pull/213.html#attribution-source-expiry says it's a point in time, rather than a duration. I think the `<a>` attribute is a duration, which would remove the timezone ambiguity if you rounded that value before converting it to an attribution source. There's still ambiguity around leap seconds, so you could say "to the nearest day (86400 seconds)".